gmm_client/test/basic_SUITE.erl

```diff
@@ -0,0 +1,102 @@
+-module(basic_SUITE).
+-include_lib("common_test/include/ct.hrl").
+-include_lib("stdlib/include/assert.hrl").
+-export([
+    all/0,
+    groups/0,
+    init_per_suite/1,
+    end_per_suite/1,
+    init_per_group/2,
+    end_per_group/2
+]).
+-export([
+    vertices_test/1,
+    edges_test/1
+]).
+
+groups() -> [{basic_flow, [sequence], [
+    vertices_test,
+    edges_test
+]}].
+
+all() -> [{group, basic_flow}].
+
+init_per_suite(Config) ->
+    [{ct_hooks, [docker_compose_cth]} | Config].
+
+end_per_suite(_Config) ->
+    ok = application:stop(gmm_client).
+
+init_per_group(basic_flow, _Config) ->
+    timer:sleep(1000), %% from unknown reasons some delay is needed before executing first tests - without it tests fail with reason {badmatch, {error, closed}}
+    {ok, _} = application:ensure_all_started(gmm_client),
+    ok;
+init_per_group(_Group, _Config) ->
+    ok.
+
+end_per_group(_Group, _Config) ->
+    ok.
+
+vertices_test(_Config)->
+    % given
+    client:add_user("Jan"),
+    client:add_group("Friends"),
+    client:add_space("Space"),
+    client:add_provider("Provider"),
+    % when
+    UserId = client_test_helper:get_id_from_the_list(),
+
+    % then
+    [
+      ?assertEqual("Jan", client_test_helper:get_vertex_name(UserId))
+    ],
+    % when
+    client:delete_vertex(UserId),
+    SpaceId = client_test_helper:get_id_from_the_list(),
+
+    % then
+    [
+      ?assertEqual("Space", client_test_helper:get_vertex_name(SpaceId))
+    ],
+    client:delete_vertex(SpaceId).
+
+
+edges_test(_Config)->
+  % given
+  client:add_user("Jan"),
+  client:add_user("Filip"),
+
+  JanId = client_test_helper:get_id_from_the_list(),
+  FilipId = client_test_helper:get_second_id_from_the_list(),
```

Comment:
Proponuję wywalić te pojedyńcze funkcje i zrobić coś ogólniejszego
```suggestion
  JanId = client_test_helper:get_id_from_list(0),
  FilipId = client_test_helper:get_id_from_list(1),
```

Albo w ogóle:
```suggestion
  IdsList = client_test_helper:get_ids_list(),
  JanId = lists:nth(1, IdsList),
  FilipId = lists:nth(2, IdsList),
```

Reply:
To też można załatwić pattern matchem, zamiast funkcji `lists:nth`

Reply:
owszem, ale co w sytuacji kiedy potencjalnie tych wierzchołków jest dużo? no chyba że zakładamy jakiś górny limit na ich ilość tak, że, można to zapisać ręcznie

Reply:
gotowe